source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -100,26 +138,98 @@ The ovirt-provider-ovn provides an opinionated subset of OpenStack's networking
+configure those.
 
 ## Routing
 
 Routing among IPv6 subnets is seemlessly provided by an OVN logical router
 entity.
 
-Since the scope of the feature is just to implement stateful DHCPv6, no router
-changes are required.
+OVN logical routers are also required whenever the 'slaac' or
+'dhcpv6-stateless' options are configured on the subnet, since they are what
+generates the RA messages that clients require to configure their IP addresses.
 
-If, in a given future, stateless address auto-configuration becomes a goal,
-the logical router ports have to be configured, setting its
-**ipv6_ra_configs:address_mode** to 'slaac'.
+Just creating the logical router is indeed not enough; each subnet that is
+configured to operate statelessly - either 'slaac' or dhcpv6_stateless -
+requires that very same subnet to be connected to at least one OVN logical router.
 
+That will create OVN logical flows that are responsible for generating the
+Router Advertisements messages the clients require to configure their IP addresses.
+These RA messages are generated as replies to the Router Solicitation messages.
+
+It is also possible to send those periodically, but I see no reason to do so.
 The above change would be implemented in the following way:
+
 1. when a subnet is created, it's **ipv6_address_mode** parameter is stored in
 the subnet's external ids
 2. when that subnet is attached to a router, its **ipv6_ra_configs:address_mode**
 should be set from the subnet's external ids address mode value.
 
+## Guest (VM) configuration
+
+There are three different ways to configure the guests, enabling them to have
+IPv6 addresses, each with different advantages / drawbacks.
+
+### DHCPv6
+
+As indicated on section [](), DHCPv6 is one option to configure the guests. It
+requires **dhclient** to be installed in the guest VM, and since it lacks the
+network prefix length in its message payloads, it will only correctly configure
+guest attached to networks whose prefix length is 64.
+
+This guest configuration is no longer possible in EL8, due to [bug](https://bugs.launchpad.net/ubuntu/+source/isc-dhcp/+bug/1609898).
+
+### Linux Kernel RA configuration
+
+This option does not require any additional package to be installed in the
+guest VM, and involves configuring directy the behavior in the linux kernel,
+as defined [here](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt).
+
+This configuration allows for dhcpv6-stateless configurations, where the IPv6 network prefix is sent on RA messages.
+
+To follow it, the following files have to be created, and have a '1' in them:
+- accept_ra
+- autoconf
+- accept_redirects
+- accept_ra_pinfo
+
+And that the *forwarding* flag is disabled - e.g. :
+```bash
+echo "0" > "/proc/sys/net/ipv6/conf/<YOUR_IF_NAME>/forwarding"
+```
+
+This option would work on both EL7 and EL8.
+
+### NetworkManager
+
+This option involves configuring the NetworkManager daemon - which is obviously required.
```

Comment:
'- which is obviously required' - can be removed.

Reply:
Right. Adjectives are bad :)

Done